core/controllers/resources_test.py

```diff
@@ -397,6 +397,5 @@ def test_non_matching_extensions_are_detected(self):
         )
         self.logout()
         self.assertEqual(response_dict['code'], 400)
-        self.assertIn(
-            'Although the filename extension indicates the file',
-            response_dict['error'])
+        self.assertEqual(response_dict['error'], 'Audio not recognized as '
```

Comment:
Optional: consider breaking after '(', and have the second arg be all on one line, instead of breaking the string over two lines.